tests/unit_tests/test_wave_02.py

```diff
@@ -34,7 +34,10 @@ def test_get_no_matching_items_by_category():
 
     items = vendor.get_by_category("electronics")
 
-    raise Exception("Complete this test according to comments below.")
+    #raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
+    assert item_a not in items
+    assert item_b not in items
+    assert item_c not in items
```

Comment:
Tests work better when there is no room for ambiguity.  In this case, checking that these items are not in the list of items is great, but it leaves room for there to be some other item that is also not wanted.  Adding something like:
```
    assert len(items) == 0
```
helps to make it clear that the items list should be empty.